src/input.rs

```diff
@@ -237,7 +237,7 @@ impl Action {
     fn paste<A: ActionContext>(&self, ctx: &mut A, contents: String) {
         if ctx.terminal_mode().contains(TermMode::BRACKETED_PASTE) {
             ctx.write_to_pty(&b"\x1b[200~"[..]);
-            ctx.write_to_pty(contents.into_bytes());
+            ctx.write_to_pty(contents.replace("\x1b[201~","").into_bytes());
```

Comment:
Termite just removes the `\x1b` so it would just look like `[201~`, XTerm does the same thing and URxvt does not do it at all.

So based on these I'd follow their example of removing `\x1b` to make it consistent with what users may have experienced on other terminal emulators.

Reply:
OK, changed it to remove \x1b.